rebasehelper/tests/conftest.py

```diff
@@ -59,3 +61,18 @@ def pytest_collection_modifyitems(items):
             item.add_marker(pytest.mark.functional)
         else:
             item.add_marker(pytest.mark.standard)
+
+
+@pytest.fixture
+def mocked_spec_object(monkeypatch, spec_attributes):
+    spec_attributes['save'] = lambda: None
+    spec = SpecFile.__new__(SpecFile)
+    for attribute, value in spec_attributes.items():
+        monkeypatch.setattr(spec, attribute, value, raising=False)
+        if attribute == 'spec_content' and isinstance(value, str):
+            monkeypatch.setattr(spec, attribute, SpecContent(value), raising=False)
+    try:
+        spec.tags = Tags(spec.spec_content, spec.spec_content)
+    except AttributeError:
+        pass
```

Comment:
Actually, `tags` could have been passed in, so `hasattr()` can't be avoided it seems. Sorry.